openmetadata-ui/src/main/resources/ui/src/interface/data-insight.interface.ts

```diff
@@ -27,3 +30,7 @@ export interface ChartFilter {
   startTs: number;
   endTs: number;
 }
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export interface DataInsightChartTooltipProps extends TooltipProps<any, any> {
```

Comment:
why `any`

Reply:
we don't have a type for Tooltip value and Tooltip that's why we have to use the type "any".